pmoves/tools/observability/tracing_specialist.py

```
@@ -0,0 +1,357 @@
+            lookbackHours: Lookback period in hours
+
+        Returns:
+            List of trace metadata
+        """
+        end = datetime.now()
+        start = end - timedelta(hours=lookback_hours)
+
+        params = {
+            "service": service,
+            "limit": str(limit),
+            "lookback": f"{lookbackHours}h"
```

Ruff (0.15.10)

[error] Undefined name `lookbackHours`

(F821)